Sources/MapboxMaps/Ornaments/OrnamentsManager.swift

```diff
@@ -177,23 +177,33 @@ public class OrnamentsManager: NSObject {
     }
 
     private func constraints(with view: UIView, position: OrnamentPosition, margins: CGPoint) -> [NSLayoutConstraint] {
+        // handle AutoLayout changes when device character direction changes
+        let currentDeviceLanguage: String! = Locale.current.languageCode
+        let rightToLeft = Locale.LanguageDirection.rightToLeft
+
+        if Locale.characterDirection(forLanguage: currentDeviceLanguage) == rightToLeft {
+            UIView.appearance().semanticContentAttribute = .forceRightToLeft
```

Comment:
I don't think we should force `semanticContentAttribute` to appearance proxy as it affects other views too
```suggestion
            view.superview?.semanticContentAttribute = .forceRightToLeft
```